tests/common/TestsSetup.hs

```diff
@@ -0,0 +1,42 @@
+-----------------------------------------------------------------------------
+--
+-- Module      :  Main
+-- Copyright   :  (c) Oleg Grenrus 2015
+-- License     :  MIT
+--
+-- Maintainer  :  Phil Freeman <paf31@cantab.net>
+-- Stability   :  experimental
+-- Portability :
+--
+-- |
+--
+-----------------------------------------------------------------------------
+module TestsSetup where
+
+import Data.Maybe (fromMaybe)
+import Control.Applicative
+
+import Control.Monad
+
+import Control.Applicative
+
+import Control.Monad.Trans.Maybe
+
+import System.Process
+import System.Directory
+
+findNodeProcess :: IO (Maybe String)
```

Comment:
This could be generalised and moved somewhere into shared code, @garyb opinions? seems quite the same as in `psc-publish`